astropy/coordinates/tests/test_intermediate_transformations.py

```diff
@@ -316,15 +316,15 @@ def test_itrs_topo_to_hadec_with_refraction():
 
     assert_allclose(hadec22.ha - hadec2.ha, 0 * u.mas, atol=0.1 * u.mas)
     assert_allclose(hadec22.dec - hadec2.dec, 0 * u.mas, atol=0.1 * u.mas)
-    assert_allclose(hadec22.distance - hadec2.distance, 0 * u.cm, atol=10.0 * u.cm)
+    assert_allclose(hadec22.distance - hadec2.distance, 0 * u.cm, atol=10.205 * u.cm)
 
     # Refraction removed
     itrs = hadec22.transform_to(itrs_frame)
     hadec33 = itrs.transform_to(hadec_frame1)
 
     assert_allclose(hadec33.ha - hadec3.ha, 0 * u.mas, atol=0.1 * u.mas)
     assert_allclose(hadec33.dec - hadec3.dec, 0 * u.mas, atol=0.1 * u.mas)
-    assert_allclose(hadec33.distance - hadec3.distance, 0 * u.cm, atol=10.0 * u.cm)
+    assert_allclose(hadec33.distance - hadec3.distance, 0 * u.cm, atol=17.01 * u.cm)
```

Comment:
Is the new atol ok?

Reply:
@mhvk , sure, but what about the other two tests above? I don't have a preference between rtol and atol; it depends on the intention of the comparisons. Your advice would be much appreciated!

Reply:
I think `rtol` is more logical here, so maybe try changing all three to that. But maybe test locally...  Thanks!!!

Reply:
`rtol=2e-16` was still too small. I had to increase to 2e-15 to make all 3 tests pass. See new commit. Thanks!

Reply:
I think I must have mistyped - 2e-16 is the best one could possibly hope for (1ULP); I think 2e-15 is probably what I meant...